stock-usecase/src/main/java/org/dhis2/android/rtsm/ui/home/HomeViewModel.kt

```diff
@@ -48,6 +48,9 @@ class HomeViewModel @Inject constructor(
     private val _settingsUiSate = MutableStateFlow(SettingsUiState(programUid = config.program))
     val settingsUiState: StateFlow<SettingsUiState> = _settingsUiSate
 
+    private val _orgUnitData = MutableStateFlow<OrganisationUnit?>(null)
+    val orgUnitData: StateFlow<OrganisationUnit?> = _orgUnitData
```

Comment:
You have the selected orgUnit inside the settingsUiState. Why don't you use it, instead of create a another propertie with the same information?

Reply:
Yes, makes sense. i will see this aproach